src/ContinuousEnergyNeutron.tex

```diff
@@ -1200,3 +1270,77 @@ \subsubsection{\textsf{UNR} Block}\label{sec:UNRBlock}
   \label{tab:PTableOrder}
 \end{XSSTable}
 \end{ThreePartTable}
+
+\subsubsection{\textsf{PTYPE} Block}\label{sec:PTYPEBlock}
+
+The \Block{PTYPE} is the first of the particle production blocks used for neutron and charged particle production data. These particle production blocks are given only if the number of particles \var{NTYPE} = \nxs{7} is different from zero. If the \Block{PTYPE} is present, the \Block{PTYPE} starts at the index \var{LTYPE} = \jxs{30}.
+
+{\color{red}\textbf{Question:} The particle type can be neutron, when is this actually used?}
+
+The \Block{PTYPE} gives a list of particle types for which particle production data is available. This includes cross section data (given in the {\color{red}\textbf{ADD BLOCK NAMES HERE}}), angular distribution data (given in the {\color{red}\textbf{ADD BLOCK NAMES HERE}}) and secondary particle energy distribution data (given in the {\color{red}\textbf{ADD BLOCK NAMES HERE}}).
```

Comment:
Are you planning on updating this?

Reply:
Ah, I forgot about those. I was going to pull this into the documentation because I want to update the photonuclear ACE files as well, and they do borrow a lot from these. I'll have a look.